starcheck/pcad_att_check.py

```diff
@@ -35,8 +35,18 @@ def test_check_characteristics_date():
                                     '2015Nov02 at 00:00:00.000')
     assert ok is False
 
+def recent_history(time, file):
+    for state_line in reversed(open(file).readlines()):
+        state_match = re.match('^(\d+\.\d+)\s+\|\s+(\S+)\s*$',
+                                state_line)
+        if state_match:
+            if (DateTime(state_match.group(1), format='greta').secs
+                < time):
+                return state_match.group(1), int(state_match.group(2))
+
 
 def make_pcad_attitude_check_report(backstop_file, or_list_file=None, mm_file=None,
+                                    simtrans_file=None, simfocus_file=None,
```

Comment:
If this file list expands any more we should call it `files` and make it a dict like `{'backstop': .., 'or_list': .., ..}`.  OK for now.